app/helpers/templateGenerator.js

```diff
@@ -1,119 +1,148 @@
 const fs = require('fs');
 
-const getInputFields = (params) => {
-  const paramsInputString = Object.entries(params).map(([key, value]) => {
-    const modifiedValue = typeof value === 'string'
-      ? value.replace(/&lt/g, '<').replace(/&gt;/g, '>')
-      : value;
-    return `
-      <div class="nhsuk-u-inline-block nhsuk-u-padding-4 nhsuk-u-padding-bottom-0">
-        <label for="param" class="nhsuk-grid-column-one-quarter">${key}:</label>
-        <input type="text" name="${key}" id="${key}" value="${modifiedValue}" class="nhsuk-grid-column-three-quarters nhsuk-u-font-size-16"/>
-      </div>
-    `;
-  }).join('');
-  return paramsInputString;
-};
+const generateEditableJSON = ({
+  key,
+  value,
+  isLast,
+  showKey = true,
+  isCode,
```

Comment:
Why not perhaps pass this is as blockType? which can either be `code` or `display`.
Might make the other bits further down more readable since we can just do
`options[blockType]`